docs/topics/whatsnew/whatsnew2120.md

```diff
@@ -119,7 +119,17 @@ plugin is applied on the first `executable {}` block.
 
 If you run into any issues, report them in our [issue tracker](https://kotl.in/issue) or let us know in our [public Slack channel](https://kotlinlang.slack.com/archives/C19FD9681).
 
-## Kotlin/Native: new inlining optimization
+## Kotlin/Native
+
+### Support for Xcode 16.3
+
+Starting with Kotlin 2.1.2**1**, the Kotlin/Native compiler supports Xcode 16.3 – the latest stable version of Xcode.
```

Comment:
What if we highlight the entire release version? Or would that still not be noticeable enough? 🤔